installer/ubuntu/bootstrap

```diff
@@ -55,6 +55,11 @@ fi
 # Patch debootstrap so that is does not create devices under /dev (issue #2387).
 sed -i -e 's/^setup_devices () {$/\0 return 0/' "$tmp/functions"
 
+# Patch debootstrap to use curl instead of wget
+# Note that we do not translate other parameter, and lose the progress bar, but
+# we do not use these anyway.
+sed -i -e 's/wgetprogress\(.*\) -O "$dest"/curl\1 -o "$dest"/' "$tmp/functions"
```

Comment:
curl has a `--progress-bar` option, and by default if its stdout is a tty, it'll use a more featureful progress bar

Reply:
something like `curl -# -L --connect-timeout 60 --retry 2 "$url"`...